masterlist.yaml

```
@@ -8820,6 +8820,31 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 773
 
+  - name: 'WoodworkersWhim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64507/'
+        name: 'Woodworker''s Whim'
```

You can combine the shared locations with a regex plugin, it saves a few lines:

```suggestion
  - name: 'WoodworkersWhim(_Campfire)?\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64507/'
        name: 'Woodworker''s Whim'
  - name: 'WoodworkersWhim.esp'
```